typings.json

```diff
@@ -1,5 +1,7 @@
 {
   "ambientDependencies": {
+    "chai": "github:DefinitelyTyped/DefinitelyTyped/chai/chai.d.ts#a9680ac2efd3add0d38cd0331f30ad497d9dd1e0",
+    "mocha": "github:DefinitelyTyped/DefinitelyTyped/mocha/mocha.d.ts#a9680ac2efd3add0d38cd0331f30ad497d9dd1e0",
```

Comment:
I'm curious about this one. I don't see mocha being imported anywhere (which is expected), but it's also not manually included in the build process. Does this mean that this is always injected? And does that mean that I could do `describe()` in client code, and it wouldn't error? Do we care if that's the case?